### ISSUE TITLE
Upgrading the interpreter

### ISSUE THREAD

**Issue by annie-bhaiya** [hash-define-organization/hashdefine-onlineIDE#11]
Can I work on making the compiler part available to more languages?

**Comment by irshadmd** [comment]
first, we need to fix the bugs for current languages once done we can add more languages.

**Comment by tech-gunk** [comment]
> first, we need to fix the bugs for current languages once done we can add more languages.

Can u please provide the backend code. If u really wanna hide, then please send the stderr with the response, we can show the error also.

**Comment by mdabdulshahed** [comment]
Interested in fixing this issue, what are the bugs for current language, can we discuss ?

**Comment by VaibhavPaliwal0007** [comment]
can you make the compiler part from starting?

**Comment by mdabdulshahed** [comment]
yes, i would really like to work with your guidance, if we can have meet for this we can discuss more about it
